lms/djangoapps/verify_student/views.py

```diff
@@ -946,6 +947,32 @@ def _send_email(user_id, subject, message):
     user.email_user(subject, message, from_address)
 
 
+def _set_requirement_status(attempt, namespace, status, reason=None):
+    """Set requirement status and the reason of status for given user in
+    attempt.
```

Comment:
This function sets the status of a credit requirement for the user, based on a verification checkpoint. Please explain this in the docstring, and consider naming the function more precisely.